core/pom.xml

```diff
@@ -83,12 +83,38 @@
             <artifactId>jackson-annotations</artifactId>
             <version>${jackson.version}</version>
         </dependency>
-
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <!-- Kafka's SSL utilities are used is tests. -->
```

Comment:
Nice catch! I'll address this here (in a rebased commit) and in https://github.com/confluentinc/rest-utils/pull/38 in a new commit.